modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,92 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must specify the projects associated with the {ProductShortName} in a `ServiceMeshMemberRoll` resource located in the same project as the `ServiceMeshControlPlane` resource and name it `default`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are a part of the `ServiceMeshMemberRoll` resource.
+====
+
+[NOTE]
+====
+The member projects are only updated if the {ProductShortName} control plane installation succeeds.
+====
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll`.
+
+* `ServiceMeshMemberRoll` resources are reconciled in response to the following events:
+
+** The `ServiceMeshMemberRoll` is created, updated, or deleted
+** The `ServiceMeshControlPlane` resource in the project containing the `ServiceMeshMemberRoll` is created or updated
+** A project listed in the `ServiceMeshMemberRoll` is created or deleted
+
+The `ServiceMeshMemberRoll` is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+[id="ossm-member-roll-console_{context}"]
+== Configure the member roll from the the {product-title} web console
+
+This example joins the Bookinfo project to the {ProductShortName} from the web console.
+
+.Prerequisites
+* An installed, verified {ProductName} Operator.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Click to *Home* -> *Projects*.
+
+. Click *Create Project*.
+
+. Enter a *Project Name* (for example, *bookinfo*), a *Display Name*, and a *Description*, then click *Create*.
+
+. Click *Catalog* -> *Installed Operators*.
+
+. Click the *Project* menu and choose `openshift-operators` from the list.
+
+. Click the *Istio Service Mesh Member Roll* link under *Provided APIS* for the *{ProductName}* Operator.
+
+. Click *Create Service Mesh Member Roll*.
+
+. Edit the default Service Mesh Member Roll yaml and add *bookinfo* to the *members* list.
+
+. Click *Create* to save the updated Service Mesh Member Roll.
+
+
+[id="ossm-member-roll-cli_{context}"]
+== Configure the member roll from the CLI
+
+This example joins the Bookinfo project to the {ProductShortName} from the CLI.
+
+.Prerequisites
+
+* An installed, verified {ProductShortName} Operator.
+* Location of the project where the custom resource installed the control plane.
```

Comment:
Unclear what "Location" means. Should this be "Name" instead?

Also: might not be clear what "custom resource" refers to. Should be "ServiceMeshControlPlane resource". (I'd leave out the word "custom", as it only makes sense in the context of Kubernetes/OpenShift; in the context of the service mesh, it doesn't).

Reply:
Updated to read: 

"Name of the project with the `ServiceMeshMemberRoll` resource."